src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForLoopStatement.cs

```diff
@@ -38,12 +38,15 @@ static void Main()
       Right: 
         ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 3) (Syntax: '3')
   Before:
-      IVariableDeclarationsOperation (1 declarations) (OperationKind.VariableDeclarations, Type: null) (Syntax: 'int i = 0')
-        IVariableDeclarationOperation (1 variables) (OperationKind.VariableDeclaration, Type: null) (Syntax: 'i = 0')
-          Variables: Local_1: System.Int32 i
+      IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'int i = 0')
```

Comment:
>IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'int i = 0') [](start = 6, length = 126)

I think we shouldn't have this node in the tree for this scenario. Syntactically we have VariableDeclarationSyntax, which maps to IVariableDeclarationOperation

Reply:
If, for whatever reason, we will decide to keep this node, it feel like it should be marked as implicit rather than its child.

---
In reply to: [147796009](https://github.com/dotnet/roslyn/pull/22842#discussion_r147796009) [](ancestors = 147796009)